### PR TITLE
Switch back to snapshot images in samples

### LINKED ISSUE #195: Change samples to use snapshot base images again

### PR DESCRIPTION
**Pull request by ericbottard**
Fixes #195

### CODE REVIEW

**Comment by markfisher** [comment]
we discussed this yesterday and decided to leave samples on the latest stable version (devs can always change locally)... however, we can reopen the debate if you'd like. The argument in favor of changing to snapshots is that we do have a `v0.0.1` tagged version of the riff repo: https://github.com/projectriff/riff/tree/v0.0.1

cc @trisberg

File: samples/shell/echo/echo-function.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ kind: Function
 metadata:
   name: echo
 spec:
-  protocol: stdio
+  protocol: pipes
```
Comment (markfisher) [inline samples/shell/echo/echo-function.yaml]: I think you left this here on accident

Comment (ericbottard) [reply samples/shell/echo/echo-function.yaml]: ah, yes. `git add -A samples/` :(

**Comment by trisberg** [comment]
I'd like to keep the samples at `0.0.1` for now, makes it easier to run them locally and on GKE. If I do local dev work I can change the sample. Just my 2c.

**Comment by markfisher** [comment]
closing for now

**Pull request closed without merging**